avocado/utils/nvme.py

```diff
@@ -47,7 +47,7 @@ def get_controller_name(pci_addr):
     """
     if pci_addr in pci.get_pci_addresses():
         path = f"/sys/bus/pci/devices/{pci_addr}/nvme/"
-        return os.listdir(path)
+        return "".join(os.listdir(path))
```

Comment:
some how i like older way it self as have multiple device than accessing list will be easy

Reply:
@PraveenPenguin we will have only one controller with corresponding pci_address.
each nvme drive will have specific pci_address. Hence the output is expected to be one device only.

Reply:
logs for your reference

```
[root@sys]# lspci -nnD
0563:20:00.0 Non-Volatile memory controller [0108]: Samsung Electronics Co Ltd NVMe SSD Controller PM173Xa [144d:a825]
0567:38:00.0 Non-Volatile memory controller [0108]: Samsung Electronics Co Ltd NVMe SSD Controller PM173Xa [144d:a825]
[root@sys]#
[root@sys]# pwd
/sys/bus/pci/devices/0563:20:00.0/nvme
[root@sys]# ls
nvme0
[root@sys]# 


[root@sys]# cd ../../0567\:38\:00.0/nvme/
[root@sys]#
[root@sys]# pwd
/sys/bus/pci/devices/0567:38:00.0/nvme
[root@sys]# ls
nvme1
[root@sys]#


```

Reply:
thanks, @Naresh-ibm for info, can you please address static checks